src/Controller/AdminController.ts

```diff
@@ -68,4 +71,16 @@ export class AdminController extends BaseHttpController {
 
     return this.json(result, 400)
   }
+
+  @httpPost('/users/:userUuid/subscription-token', TYPES.ApiGatewayAuthMiddleware)
```

Comment:
Hmm why do you need the `ApiGatewayAuthMiddleware` on this one?

Reply:
Right, it is not needed.